src/Modules/OrchardCore.Commerce/Abstractions/IStripePaymentService.cs

```diff
@@ -25,6 +26,16 @@ public interface IStripePaymentService
     /// </summary>
     Task<PaymentIntent> GetPaymentIntentAsync(string paymentIntentId);
 
+    /// <summary>
+    /// Calculates payment amount based on the given <paramref name="defaultTotalValue"/> and <paramref name="currencyType"/>.
+    /// </summary>
+    long GetPaymentAmount(decimal defaultTotalValue, string currencyType);
```

Comment:
I don't like these argument names, because they describe how the method is currently used but without knowing that context they are pretty mysterious.  Why not just pass in one `Amount total` argument? I don't think there is any possible scenario where these two shouldn't be tied together. Also you are already using `Amount` for the other method.